bot/cogs/tags.py

```diff
@@ -92,7 +92,18 @@ def _get_suggestions(self, tag_name: str, thresholds: Optional[List[int]] = None
         """Show all known tags, a single tag, or run a subcommand."""
         await ctx.invoke(self.get_command, tag_name=tag_name)
 
-    async def _get_command(self, ctx: Context, tag_name: TagNameConverter = None) -> None:
+    async def get_command(self, ctx: Context, tag_name: str = None) -> bool:
```

Comment:
This isn't a command so it should not be named as such. `display_tag` is better, for starters.